spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -414,7 +414,7 @@ def logger
     context 'on el-5' do
       let(:platform) { Beaker::Platform.new('el-5-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-5\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-5.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
Ok, you convinced me to do the right thing and add some tests for `install_package_with_rpm`. :)